content/references/rippled-api/admin-rippled-methods/status-and-debugging-methods/peers.md

```diff
@@ -39,114 +39,136 @@ An example of a successful response:
-        "public_key": "n9JySgyBVcQKvyDoeRKg7s2Mm6ZcFHk22vUZb3o1HSosWxcj9xPt",
-        "uptime": 99625,
-        "version": "rippled-0.30.1"
+        "address": "[::ffff:94.237.45.66]:51235",
+        "complete_ledgers": "51815004 - 51815131",
+        "ledger": "99A1E29C9F235DCCBB087F85F11756BECA606A756C22AB826AB1F319C470C3E3",
+        "load": 202,
+        "metrics": {
+          "avg_bps_recv": "18258",
+          "avg_bps_sent": "1903",
+          "total_bytes_recv": "1184272",
+          "total_bytes_sent": "65101"
+        },
+        "public_key": "n9Lg83FYh8YDivG9TcgXhq5Y3PwunmRqVfvibd19Ko9uu3DtqLBM",
+        "uptime": 2,
+        "version": "rippled-1.3.1"
       },
       {
-        "address": "72.251.233.165:51235",
-        "complete_ledgers": "18827949 - 18828973",
-        "latency": 40,
-        "ledger": "50A2577CE6EB8A92847C443BDA45F5C5F0A22B9C6F4B47DBA0C12BDA75001D01",
-        "load": 63,
-        "public_key": "n9M77Uc9CSaSFZqt5V7sxPR4kFwbha7hwUFBD5v5kZt2SQjBeoDs",
-        "uptime": 99625,
-        "version": "rippled-0.30.1"
+        "address": "[::ffff:149.56.21.37]:51235",
+        "complete_ledgers": "51478129 - 51815129",
+        "ledger": "462FA0B34723C12EBE4DC9974B397D6A21D4F1770DE3CD584D4E33DBC83FC247",
+        "load": 182,
+        "metrics": {
+          "avg_bps_recv": "132648",
+          "avg_bps_sent": "13935",
+          "total_bytes_recv": "3983628",
+          "total_bytes_sent": "433757"
+        },
+        "public_key": "n94Dutms7xoSgWjbjYftDJGXvc5jaeYZq3J7o1Jwo35dZAyuzNWT",
+        "uptime": 14,
+        "version": "rippled-1.3.1"
       },
       {
-        "address": "72.251.232.173:51235",
-        "complete_ledgers": "32570 - 18828973",
-        "latency": 40,
-        "ledger": "50A2577CE6EB8A92847C443BDA45F5C5F0A22B9C6F4B47DBA0C12BDA75001D01",
-        "load": 71,
-        "public_key": "n9JveA1hHDGjZECaYC7KM4JP8NXXzNXAxixbzcLTGnrsFZsA9AD1",
-        "uptime": 99625,
-        "version": "rippled-0.31.0-b6"
+        "address": "77.117.40.191:51235",
+        "complete_ledgers": "51812267 - 51815071",
+        "latency": 30555,
+        "ledger": "0562C7A1898196FDC60B8DB1838961BFBA6B9304B1510E6030B32A4FB38400FB",
+        "load": 3783,
+        "metrics": {
+          "avg_bps_recv": "495673",
+          "avg_bps_sent": "57527",
+          "total_bytes_recv": "17137378",
+          "total_bytes_sent": "4140188"
+        },
+        "public_key": "n9LELaTdzKwfzhvHZKKrD3ZEJtWABSg4ocAT4dz5Vg5JgAVGwVdd",
+        "uptime": 73,
+        "version": "rippled-1.4.0"
       },
       {
-        "address": "98.167.120.212:51235",
-        "complete_ledgers": "18828845 - 18828973",
-        "latency": 99,
-        "ledger": "50A2577CE6EB8A92847C443BDA45F5C5F0A22B9C6F4B47DBA0C12BDA75001D01",
-        "load": 60,
-        "public_key": "n9LDBRoqPYY7RdkNXbX1dqZXVtUKcSqzs2CZPhTH7ymA9X7Xzmpj",
-        "uptime": 99625,
-        "version": "rippled-0.30.1-rc4"
+        "address": "[::ffff:94.237.49.50]:51235",
```

Comment:
Would there be any benefit to masking these IPs?

Reply:
I don't think it's necessary. These IPs are probably obtainable just by connecting any server to the peer-to-peer network and listening to the protocol messages.